composer.json

```diff
@@ -15,11 +15,17 @@
   "require": {
     "silverstripe/framework": "^4.2",
     "silverstripe/cms": "^4.2",
-    "symbiote/silverstripe-queuedjobs": "^4.2",
-    "guzzlehttp/guzzle": "^6.3"
```

Comment:
Why was guzzle removed as a dependency? It (or at least PSR) is explicitly used by this module.